### PR TITLE
Add Bionic Spec-Op profession.

### PR DESCRIPTION
**Pull request by Noctifer-de-Mortem**
This is my first pr so tell me if I did anything wrong. This profession is a bionic version of the Special Operator with a hint of the Bionic Soldier. If there were Bionic Soldiers why not Bionic Special Operators? This profession would also be added to helicopter crash scenario.

### CODE REVIEW

File: data/json/professions.json [hunk 1]
```diff
@@ -2209,6 +2209,71 @@
         }
     },
     {
+        "type": "profession",
+        "ident": "bio_specops",
+        "name": "Bionic Spec-Op",
+        "description": "You were the latest in the military cyborg program. A bionic special forces operator capable of handling any combat scenario, that is why you are still alive as you killed your former allies. As far as you can tell, military command abandoned you in this hellhole when you missed the emergency evac.",
```
Comment (Night-Pryanik) [inline data/json/professions.json]: Double spaces after periods, please.

**Comment by a-v-e** [comment]
Laser rifle,  lots of useful bionics, and all this for meager 7 points? Too OP.

**Comment by Noctifer-de-Mortem** [comment]
> Laser rifle, lots of useful bionics, and all this for meager 7 points? Too OP.
How many points do you suggest then?

**Comment by ave-** [comment]
Laser rifle is a `lab finale` item, it is considered to be a reward for advanced survivors, so getting it from the start is a bit of a cheat. I'd say, just remove the rifle entirely, as even 20 points is not enough for that.

**Comment by Noctifer-de-Mortem** [comment]
How about it being replaced with the pistol variant under the guise the main rifle was lost and you are down to your backup?

**Comment by ave-** [comment]
Sorry to disappoint you, but v29 laser pistol is lab finale too.

**Comment by Noctifer-de-Mortem** [comment]
Then I would just drop the rifle and say it was lost in the chaos in the description.

**Comment by Cyrano7** [comment]
"that is why you are still alive as you killed your former allies." I don't quite get this part. Why did you kill them? Did you have to? Did they become zombies?

**Comment by Mecares** [comment]
perhaps adding an "after they turned" to the line to at least imply that they were Zombies

**Comment by Cyrano7** [comment]
How about "A bionic special forces operator equipped with a laser rifle that feeds from your internal power supply... or you did until you lost it along with your squad during the initial chaos."

File: data/json/professions.json [hunk 2]
```diff
@@ -2209,6 +2209,75 @@
         }
     },
     {
+        "type": "profession",
+        "ident": "bio_specops",
+        "name": "Bionic Spec-Op",
+        "description": "You were the latest in the military cyborg program.  A bionic special forces operator equiped a laser rifle that feeds from your internal power... or you did until you lost it during the initial chaos along with your squad.  With a scavanged weapon and ammo from the fallen you fight on.",
```
Comment (Cyrano7) [inline data/json/professions.json]: "equiped" should be equipped.

**Comment by Cyrano7** [comment]
Why even mention the laser rifle if you are going to give the profession another weapon.

**Comment by Noctifer-de-Mortem** [comment]
> Why even mention the laser rifle if you are going to give the profession another weapon.

Justification for the ups bionic.

**Comment by Finalsong** [comment]
Thinking that they should have the same outfit as a normal spec-ops or bionic soldier. That is, no helmet and no mask. Or dump pouches, etc.

**Pull request closed without merging**